mne/preprocessing/ica.py

```diff
@@ -2302,9 +2306,9 @@ def _plot_corrmap(data, subjs, indices, ch_type, ica, label, show, outlines,
 
 
 @verbose
-def corrmap(icas, template, threshold="auto", label=None,
-            ch_type="eeg", plot=True, show=True, verbose=None, outlines='head',
-            layout=None, sensors=True, contours=6, cmap='RdBu_r'):
+def corrmap(icas, template, threshold="auto", label=None, ch_type="eeg",
+            plot=True, show=True, verbose=None, outlines='head', layout=None,
+            sensors=True, contours=6, cmap='RdBu_r'):
```

Comment:
I think my role in this code adding was mostly cleaning it up to get it merged so I don't know actually. `cmap=None` with corresponding docstring was in a recent PR, though, so that sounds right to me, too.